app/pages/onboarding/03-restore-wallet/restore-wallet.tsx

```diff
@@ -40,11 +40,13 @@ export const RestoreWallet: React.FC = () => {
       return;
     }
 
-    if (!validateMnemonic(mnemonic)) {
+    const parsedMnemonic = mnemonic.toLowerCase();
+
+    if (!validateMnemonic(parsedMnemonic)) {
       setError('Not a valid bip39 mnemonic');
       return;
     }
-    dispatch(persistMnemonic(mnemonic.trim()));
+    dispatch(persistMnemonic(parsedMnemonic.trim()));
```

Comment:
Should we just trim it before validation, too? To me, I like auto-fixes because I can't think of a reason a user would intentionally add whitespace, or enter an invalid bip39 phrase.

Reply:
yep, good point